lib/src/components/NxVulnerabilityDetails/NxVulnerabilityDetails.scss

```diff
@@ -131,9 +118,13 @@
     }
 
     blockquote {
-      border-left: 5px solid $nx-grey-2;
-      margin: 0 0 20px;
-      padding: 0 0 0 15px;
+      @include container-vertical();
+      background-color: #f4f5f9;
+      border: 1px solid #d1d6f0;
+      border-left-width: 8px;
+      border-radius: $nx-border-radius;
+      margin: 0 0 $nx-spacing-l 0;
+      padding: $nx-spacing-l;
```

Comment:
A lot of duplication between this and `pre`

Reply:
> A lot of duplication between this and `pre`; could be reduced with a mixin or an `@extends`

Probably not exactly what you were thinking, but I think it's clear: https://github.com/sonatype/sonatype-react-shared-components/pull/129/commits/c4bdee8cb4e1e1f8ad243b06cca39aa053fb1ec1

Reply:
> > A lot of duplication between this and pre; could be reduced with a mixin or an @extends
>
> Probably not exactly what you were thinking, but I think it's clear: c4bdee8

That doesn't reduce the duplication, the styles are still present twice (now in `blockquote` and in `%nx-pre`)

Reply:
> > > A lot of duplication between this and pre; could be reduced with a mixin or an @extends
> > 
> > 
> > Probably not exactly what you were thinking, but I think it's clear: [c4bdee8](https://github.com/sonatype/sonatype-react-shared-components/commit/c4bdee8cb4e1e1f8ad243b06cca39aa053fb1ec1)
> 
> That doesn't reduce the duplication, the styles are still present twice (now in `blockquote` and in `%nx-pre`)

I really don't mind the duplication between `pre` and `blockquote` they are very different elements. If the thick left border style starts popping up more frequently then we could make up a mixin for it but right now we have just these two instances. I see no point in having them share spacing values (in an extend or a mixin) there's too much risk of a designer changing those.